pom.xml

```diff
@@ -822,5 +822,27 @@ THE SOFTWARE.
         <failIfNoTests>false</failIfNoTests>
       </properties>
     </profile>
+    <profile>
+      <id>jdk-17-and-above</id>
+      <activation>
+        <jdk>[17,)</jdk>
+      </activation>
+      <properties>
+        <!--
+          Animal Sniffer does not work on Java 17, but we cannot remove it from our configuration
+          entirely until we drop support for Java 8. As a temporary solution, we skip it when
+          running on Java 17. When JENKINS-54842 is resolved, this can be deleted.
+        -->
+        <animal.sniffer.skip>true</animal.sniffer.skip>
+
+        <!--
+          Spotless requires custom arguments to be added to .mvn/jvm.config when running on Java 17,
+          but we cannot add those custom arguments until we drop support for Java 8. As a temporary
+          workaround, we skip Spotless when running on Java 17. When JENKINS-68015 is resolved, this
```

Comment:
https://issues.jenkins.io/browse/JENKINS-68015 :eyes: workaround not currently applied in https://github.com/jenkinsci/pom/pull/209?

Reply:
The workaround referenced in [JENKINS-68015 (comment)](https://issues.jenkins.io/browse/JENKINS-68015?focusedCommentId=422939&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-422939) would need to be applied to _this_ repository's `.mvn/jvm.config`, not the parent POM. But doing that in this repository is not feasible as long as we continue to support building with Java 8.